app/src/main/java/com/mapbox/maps/testapp/examples/JavaInterfaceChecker.java

```diff
@@ -22,6 +22,8 @@
 import android.util.AttributeSet;
 import android.view.Surface;
 
+import androidx.annotation.NonNull;
+
```

Comment:
nit: empty line

Reply:
this was autoformatted, so I guess it's fine 😄 it seems like a pattern when the prefix of the import changes, the new line will be added to separate them.